client/app/widgets/Notifications/__tests__/Notifications.spec.jsx

```diff
@@ -0,0 +1,76 @@
+// @flow
+import React from 'react';
+import axios from 'axios';
+import { mount } from 'enzyme';
+import { Notifications } from '../index';
+
+let axiosGetSpy;
+let axiosDeleteSpy;
+let componentDidMountSpy;
+
+const data = {
+  signed_in: 1,
+  fetch_notifications: ['Notification 1'],
+};
+
+const component = (
+  <Notifications
+    element={<button type="button">Notifications</button>}
+    plural="Notifications"
+    none="There are none"
+    clear="Clear"
+  />
+);
+
+describe('Notifications', () => {
+  beforeEach(() => {
+    axiosGetSpy = jest.spyOn(axios, 'get').mockImplementation(() => Promise.resolve({
+      data,
+    }));
+    axiosDeleteSpy = jest.spyOn(axios, 'delete').mockImplementation(() => Promise.resolve({
+      data: { ok: true },
+    }));
+    componentDidMountSpy = jest
+      .spyOn(Notifications.prototype, 'componentDidMount')
+      .mockImplementation(() => {});
```

Comment:
Could you explain why you decided to mock out `componentDidMount()` and not have it call `fetchNotifications()`? I think it should be enough to use your Axios spies above. It's a general best practice to use mocks only when necessary.